package.json

```diff
@@ -18,7 +18,7 @@
     "jsonwebtoken": "^8.5.1",
     "lodash": "^4.17.21",
     "mongoose": "^6.2.7",
-    "next": "^12.3.3",
+    "next": "^12.3.4",
```

Comment:
please revert this! We do not want to update the package. as of now

Reply:
Sure !! @anmode